plone/app/locales/locales/sl/LC_MESSAGES/plone.po

```diff
@@ -14907,19 +14907,19 @@ msgstr "Vsebinska pravila so globalno onemogočena. Nobeno pravilo se ne bo akti
 
 #: plone.formwidget.recurrence/plone/formwidget/recurrence/browser/i18n.py:21
 msgid "weekly_interval_1"
-msgstr ""
+msgstr "Ponavlja se:"
 
 #: plone.formwidget.recurrence/plone/formwidget/recurrence/browser/i18n.py:22
 msgid "weekly_interval_2"
-msgstr ""
+msgstr "teden(ov)"
 
 #: plone.formwidget.recurrence/plone/formwidget/recurrence/browser/i18n.py:23
 msgid "weekly_weekdays"
-msgstr ""
+msgstr "Ponavlja se:"
 
 #: plone.formwidget.recurrence/plone/formwidget/recurrence/browser/i18n.py:24
 msgid "weekly_weekdays_human"
-msgstr ""
+msgstr " "
```

Comment:
is the space by intend? (more occurrences below)

Reply:
@jensens yes, there is a couple of instances like that. It is intentional.